src/main/java/com/google/devtools/build/lib/packages/BuildType.java

```diff
@@ -70,6 +70,10 @@ public final class BuildType {
       LabelKeyedDictType.create(Type.STRING);
   /** The type of a list of {@linkplain #LABEL labels}. */
   @SerializationConstant public static final ListType<Label> LABEL_LIST = ListType.create(LABEL);
+  /** The type of a dictionary of {@linkplain #LABEL_LIST label lists}. */
```

Comment:
Please split the new attribute type (and related formatter/util/etc) into a separate CL: this is going to take some care and tests to integrate with the internal codebase and I want to be able to handle it separately.